ui/src/reducers/index.js

```diff
@@ -11,12 +18,31 @@ const columns = [
 
 const apps = { rows: [], data: {} };
 
+const appDetailColumns = [
+  { title: 'App Version', transforms: [ sortable ] },
+  { title: 'Current Installs', transforms: [ sortable ] },
+  { title: 'Launches', transforms: [ sortable ] },
+  { title: 'Last Launched', transforms: [ sortable ] },
+  { title: 'Disable on Startup', transforms: [ sortable ] },
+  { title: 'Custom Disable Message', transforms: [ sortable ] }
+];
+
+const appDetailRows = [
+  [ 'v1.0', '25', '321', '2019-01-01 09:55:10', true, 'Deprecated. Please upgrade to latest version' ],
+  [ 'v1.1', '55', '621', '2019-01-11 10:45:03', true, 'Deprecated. Please upgrade to latest version' ],
+  [ 'v1.2', '75', '921', '2019-01-20 12:12:12', false, 'LTS' ],
+  [ 'v1.3', '125', '1221', '2019-01-31 11:05:43', false, 'Curent version' ],
+  [ 'v1.5', '40', '120', '2019-02-15 10:02:50', false, 'Beta version' ]
+];
+
 const sortBy = { direction: SortByDirection.asc, index: 0 };
 
 const initialState = {
   apps: apps,
```

Comment:
think we should probably rename `apps` & some of the other names here for consistency now that we have appDetails data too

Reply:
I was somewhat following naming convention that a REST api would use. Apps for the list of apps and app for a singular app. There should only ever be one app stored in Redux for the state of the detailed app page.